### PR TITLE
Add read bytestream data example code 

### PR DESCRIPTION
**Pull request by mbthiery**

### CODE REVIEW

File: docs/oracles/oracle-data.mdx [hunk 1]
```diff
@@ -112,13 +112,125 @@ find the correct protobuf definition.
...
+that as a big endian u32. Then use that u32 number as the number of subsequent bytes to read. Decode
+those subsequent bytes using the associated proto definition. Repeat this process until you've read
+the entire file.
 
-Alternatively you can adapt
-[this sample oracle observer repo](https://github.com/dewi-alliance/oracle-observer-sample/tree/main/oracle-ingestor-lambda)
-for your use-case.
+### Javascript Example
+
+```
```
Comment (jthiller) [inline docs/oracles/oracle-data.mdx]:
```suggestion
```js
```

File: docs/oracles/oracle-data.mdx [hunk 2]
```diff
@@ -112,13 +112,125 @@ find the correct protobuf definition.
...
+    offset += messageLength + 4;
+  }
+});
+```
+
+### Rust Example
+
+This leverages existing Rust dependencies used by the oracles.
+
+```
```
Comment (jthiller) [inline docs/oracles/oracle-data.mdx]:
```suggestion
```rust
```

**Comment by deasydoesit** [comment]
If we add the following to `docusaurus.config.js`, we can get rust and protobuf code theming in `oracle-data.mdx` by adding `rust` and `protobuf` flags to the respective code blocks.
```
    prism: {
      ...
      additionalLanguages: ['rust', 'protobuf'],
    }, 
```

**Comment by mbthiery** [comment]
@deasydoesit nice find - added toml as well

**Review by deasydoesit — APPROVED** [review]

**Review by jthiller — APPROVED** [review]

**Pull request merged**